<!--igvc_run.launch-->
<launch>


  <node pkg="tf" type="static_transform_publisher" name="map_to_bb2"
        args="0 0 0.53 0 0 0 map bumblebee2 100" />

  <node pkg="usma_triclops" type="camera_system" name="camera_system" output="screen"/>
  <node pkg="usma_triclops" type="line_filter" name="line_filter" output="screen"/>
  <node pkg="usma_triclops" type="vision3d" name="vision3d" output="screen">
    <param name="triclopsMode" value="3" type="int"/>
  </node>
  
  
<!--  <node pkg="image_view" type="image_view" name="Left_line_image" output="screen" args="image:=/camera/left/linefiltered" />
  <node pkg="image_view" type="image_view" name="Left_rgb_image" output="screen" args="image:=/camera/left/rgb" />

rosrun image_view image_view image:=/camera/left/linefiltered
rosrun image_view image_view image:=/camera/left/rgb
 Start an rviz node with a custom configuration
<node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find usma_triclops)/launch/triclops_camera.rviz"/>
-->
</launch>

